pkg/mdm/mdmtest/apple.go

```diff
@@ -22,6 +22,7 @@ import (
 	"time"
 
 	"github.com/fleetdm/fleet/v4/pkg/fleethttp"
+	sharedMdm "github.com/fleetdm/fleet/v4/pkg/mdm"
```

Comment:
Nit: I think just for consistency with other renamed package imports we should go with `shared_mdm` (or `sharedmdm` I guess, it's not _that_ consistent already).